### DISCUSSION TITLE
Embed dashboard iframe is blocked by uBlock

### DISCUSSION THREAD

**Discussion by bdcorps** [plausible/analytics#3073] [Anything Else]
I am trying to embed site analytics into my Next.js app via the Embed link as an iframe. It doesn't load because uBlock is blocking the iframe by detecting the `plausible.io` domain. I'm using Plausible Cloud.

https://github.com/plausible/analytics/assets/6132555/0e0523a7-3f67-49df-8079-8df3491cb0b6

How I'm using the iframe in my Next.js component:
```
<iframe
        title="Analytics"
        plausible-embed="true"
        src={`https://plausible.io/share/sukh4.engyne.app?auth=vod7TUuFlcnqE5F7xMQU0&embed=true&theme=light&background=transparent`}
        scrolling="no"
        loading="lazy"
        style={{ width: "1px", minWidth: "100%", height: "1800px" }}
      ></iframe>
```

I've tried adding a rewrite for `/share` but this doesn't help either.
```
const nextConfig = {
  reactStrictMode: true,
  swcMinify: true,
  async rewrites() {
    return [
      {
        source: '/stats/js/script.js',
        destination: 'https://plausible.io/js/script.js'
      },
      {
        source: '/stats/api/event',
        destination: 'https://plausible.io/api/event'
      },
      {
        source: '/stats/js/host.js',
        destination: 'https://plausible.io/js/embed.host.js'
      },
      {
        source: '/share',
        destination: 'https://plausible.io/share'
      }
    ];
  },
}

module.exports = nextConfig
```

Any way you have fixed this?

**Comment by metmarkosaric** [comment]
hi @bdcorps! Easyprivacy maintainers have recently decided that they want to block this type of website content even though it doesn't include any trackers and it worsens the user experience so the best way to get around it is to disable the adblocker on the specific site or whitelist that site in your adblocker settings

**Reply by bdcorps** [reply]
Thank you for your reply @metmarkosaric! Since I'm providing the dashboard to my clients as a white labelled solution, disabling the ad blocker might not be feasible. Are there any plans to fix this on the roadmap? A few ways I can see this fixed:

1. It looks like if I ask for the underlying API data from Plausible using a proxy, that data does come back correctly. Using this, you could move all the UI components for the share dashboard into a React component (maybe an NPM package) that folks can implement in their apps.
2. uBlock seems to block the embed because of the plausible.io domain, is there a way to host the host script on a custom domain and then call it?
